### PR TITLE
Adbp 2383

### PR DESCRIPTION
**Pull request by usamabutt101**
### Why
PFM stream
Please give a short description why these code changes are being made, by replacing this line.

Link to Jira ticket: <place here>

### ToDo

- [ ]
- [ ]
- [ ]

If this is a `Draft:` or `WIP:` PR, please describe still needed actions, else delete section

### CODE REVIEW

File: src/context/onboarding-context.ts [hunk 1]
```diff
@@ -953,81 +967,108 @@ export function useAccountOriginationContextValue(): AccountOriginationContextDa
...
             {
-              addressType: 'Mailing Address',
+              addressType: "Mailing Address",
               line1: val?.addressLine,
               line2: val?.addressLine2,
               state: val?.state,
               postcode: val?.postcode,
               city: val?.city,
-              country: 'Malaysia',
+              country: "Malaysia",
```
Comment (sharmal33) [inline src/context/onboarding-context.ts:976]: why this one is hardcode?

File: src/context/onboarding-context.ts [hunk 2]
```diff
@@ -953,81 +967,108 @@ export function useAccountOriginationContextValue(): AccountOriginationContextDa
...
-              }
-            })
-          : null;
-          setAnnouncement(mostPriority ? mostPriority.nextActionCode : '')
+  const getAnnouncements = useCallback(
+    async (onError?: (error: Error) => void) => {
+      try {
+        const response = await onboardingService.getAnnouncements();
+        if (response) {
+          console.log("DATA ANNOUNCEMENT::::::::", response.data);
```
Comment (sharmal33) [inline src/context/onboarding-context.ts:1035]: remove logs

File: src/context/onboarding-context.ts [hunk 3]
```diff
@@ -374,47 +378,61 @@ export function useAccountOriginationContextValue(): AccountOriginationContextDa
...
+        if (
+          application.status === "Review" &&
+          application.productDetails.productId === "SavingAccount-i"
+        ) {
+          const withStatisfyStatus = application.applicationStatuses.find(
+            (status: any) => {
+              return (
+                status.statusName === "EDD" &&
+                status.statusValue === "Pending Submission"
+              );
```
Comment (sharmal33) [inline src/context/onboarding-context.ts:391]: Please get all static values from const file

**Review by sharmal33 — CHANGES_REQUESTED** [review]
Please do the requested changes

File: src/context/onboarding-context.ts (hunk 1 above)
Comment (usamabutt101) [reply src/context/onboarding-context.ts:976]: It's not from my work but let me check.

**Pull request closed without merging**